contest/contributions/trendecon.yml

```diff
@@ -0,0 +1,27 @@
+# Contribution's title, author, abstract
+title: Daily economic indicators based on Google searches # title case
+author: Angelica Becerra, Vera Z. Eichenauer, Ronald Indergand, Stefan Legge,
+  Isabel Martinez, Nina Mühlebach, Furkan Oguz, Christoph Sax,
+  Kristina Schuepbach, Severin Thöni, Uwe Thümmel
```

Comment:
We would typically prefer up to 4 names listed as `author`, possibly using et al., for a better integration in the [CovidR gallery](https://milano-r.github.io/erum2020-covidr-contest/). This is not strictly required though, I understand you are trying to be fair to the many contributors.

Reply:
Perhaps: 'trendecon team'? We have the contributors on the gist.

Reply:
Sounds like a good choice, I recommend to include the name(s) of the person potentially presenting the project at the pre-conference event, you can use stg like
```yaml
authors: Angelica Becerra et al., trendecon team
```